label_studio/tasks/serializers.py

```diff
@@ -217,17 +217,29 @@ def to_internal_value(self, data):
 
         return ret
 
-    @staticmethod
-    def get_completed_by_id(annotation, default=None):
-        completed_by = annotation.get('completed_by', None)
-        # user id as is
-        if completed_by and isinstance(completed_by, int):
-            return completed_by
-        # user dict
-        if completed_by and isinstance(completed_by, dict):
-            return completed_by.get('id')
-
-        return default
+    def _insert_valid_completed_by_id_or_raise(self, annotations, members_email_to_id, members_ids, current_user):
+        for annotation in annotations:
+            completed_by = annotation.get('completed_by')
+            # no completed_by info found - just skip it, will be assigned to the user who imports
+            if completed_by is None:
+                annotation['completed_by_id'] = current_user.id if current_user else None
```

Comment:
if current_user else None --- maybe project.created_by? I'm not sure we process completed_by = None in all places correctly..